config/config.exs

```diff
@@ -14,7 +14,8 @@ config :research_resource,
   qualtrics_mailinglist_id: System.get_env("QUALTRICS_MAILINGLIST_ID"),
   qualtrics_survey_id: System.get_env("QUALTRICS_SURVEY_ID"),
   qualtrics_distribution_id: System.get_env("QUALTRICS_DISTRIBUTION_ID"),
-  id_prefix: "TTRR"
+  id_prefix: "TTRR",
+  email: "tt.research.resource@gmail.com"
```

Comment:
Really good to do this 👍 @SimonLab, I'm taking out these variables in my current project refactor